src/core/render/compiler/image.js

```diff
@@ -34,6 +34,10 @@ export const imageCompiler = ({ renderer, contentBase, router }) =>
       attrs.push(`id="${config.id}"`);
     }
 
+    if (config.loading) {
+      attrs.push(`loading="${config.loading}"`);
+    }
+
```

Comment:
I found that the `loading` attr is only works on specific browsers and version, we may need mention it on docs in case of users misunderstanding the behaviors in different browsers.

Reply:
FYI.Chrome seems support it since Chrome75.
Firefox support it recently, it is unable by default tho (`dom.image-lazy-loading.enabled = false`).

Reply:
If the browser does not support lazy-loading then the attribute will be ignored and images will load immediately, as normal.